src/xplist.c.orig

```diff
@@ -0,0 +1,628 @@
+/*
```

Comment:
Good catch, thanks! This PR has been superseded by #76, which shouldn't have this file :)